configure_test.py

```diff
@@ -0,0 +1,30 @@
+import sys
+
+import yaml
+
+# Change the value of the skip flag depending on the repository used
+# repository:
+# value = "bioformats2raw", "ome_zarr", "omero_ms_zarr"
+ref_value = "ome_zarr"
```

Comment:
I don't really understand the comment here nor why `ome_zarr` is the "ref_value".

Reply:
I need to go back to that. This might have been done when evaluating options